stable/postgresql/README.md

```diff
@@ -212,6 +215,25 @@ With NetworkPolicy enabled, traffic will be limited to just port 5432.
 For more precise policy, set `networkPolicy.allowExternal=false`. This will only allow pods with the generated client label to connect to PostgreSQL.
 This label will be displayed in the output of a successful install.
 
+## Deploy chart using Docker Official PostgreSQL image
+
+From chart version 4.0.0, it is possible to use the stable/postgresql chart with the Docker Official PostgreSQL image.
```

Comment:
it is possible to use the stable/postgresql chart -> this sounds a bit odd, shouldn't we say "it is possible to use this chart"

image -> Image

Do you want to link to the image page on Docker Hub?